dev-support/ci/selective_ci_checks.bats

```diff
@@ -190,14 +190,14 @@ load bats-assert/load.bash
 }
 
 @test "native test in other module" {
-  run dev-support/ci/selective_ci_checks.sh 822c0dee1a
+  run dev-support/ci/selective_ci_checks.sh 3115158830483c527fb978e1edc15473d81191d2
```

Comment:
@adoroszlai this will fail once the PR is merged. We would have to change this to the to be commit-Id. Its a chicken and the egg problem here. FYI

Reply:
Why do you need to change this?

Reply:
We have renamed all Native integration test classes.

Reply:
Why?

Reply:
The class definition has changed. We needed to add another parameter to create linked buckets for the same tests instead of a direct bucket. We wanted to run the same set of TestOmSnapshot & TestOmSnapshotFileSystem with linked buckets. We already had one class with nativeLib & one without nativeLib. Since we added another parameter, the number of classes have doubled running it with all possible combinations.

Reply:
I have reverted the rename of the classes. I just added new class files withLinkedBuckets suffix. It could be understood without the suffix wouldn't be creating linked buckets.

Reply:
Thanks.  I was going to suggest exactly that.

Fixing the test is difficult, because we need the git tree as of the commit, but need `dev-support/ci` in its current state (because outputs also change sometimes, and the test should validate current outputs).  So we'd need to store `dev-support/ci` temporarily, `git checkout` old state, overwrite old `dev-support/ci` (which may even be absent), then restore after the test.  Not cheap and potentially dangerous for dev local changes (when run locally, not in CI).